blueocean-pipeline-api-impl/src/test/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineNodeTest.java

```diff
@@ -84,6 +84,72 @@ public void nodesTest1() throws IOException, ExecutionException, InterruptedExce
         }
     }
 
+    //JENKINS-39203
+    @Test
+    public void stepStatusForUnstableBuild() throws Exception{
```

Comment:
@michaelneale it checks individual step's status as it should be, thats all steps are successful but the run's status is UNSTABLE.